### PR TITLE
10 feature 7 cart page

### PR DESCRIPTION
**Pull request by MarkianV**
Fixed current user so it is no longer in User. Changed CartPage to match the changes.

### CODE REVIEW

**Review by danielqiuu — APPROVED** [review]

**Review by liu-samuel — APPROVED** [review]
lgtm

**Review by helenastra — APPROVED** [review]
looks good

File: src/main/java/UI/CartPage.java [hunk 1]
```diff
@@ -23,9 +24,9 @@ public class CartPage extends Page implements ActionListener {
 
 
     public CartPage() {
-        super(User.getCurrentUser().getUsername() + "'s Cart");
+        super(Main.getCurrentUser().getUsername() + "'s Cart");
```
Comment (helenastra) [inline src/main/java/UI/CartPage.java:27]: Maybe instead of the User's username's Cart, you can use their name?

Comment (lukecheseldine) [reply src/main/java/UI/CartPage.java:27]: what do you mean by this? we only store username, so we can't display their actual name. furthermore, i'm ok with having "Cart" in the title. i think the way mark is doing it now is good.

**Review by lukecheseldine — APPROVED** [review]
great job mark, thanks for removing the duplicate currentUser stuff!

**Pull request merged**